api/base/settings/defaults.py

```diff
@@ -306,14 +306,16 @@
 # Store yearly indices for time-series metrics
 ELASTICSEARCH_METRICS_DATE_FORMAT = '%Y'
 
+WAFFLE_CACHE_NAME = 'waffle_cache'
+STORAGE_USAGE_CACHE_NAME = 'storage_usage'
+
+
 CACHES = {
-    'default': {
+    STORAGE_USAGE_CACHE_NAME: {
         'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
         'LOCATION': 'osf_cache_table',
     },
-    'waffle_cache': {
+    WAFFLE_CACHE_NAME: {
         'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
     },
```

Comment:
since you're overriding caches, I think you're going to need to explicitly define the default cache here, adding:
```
 'default': {
        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
    }
```